lychee-bin/src/formatters/response/color.rs

```diff
@@ -65,14 +88,18 @@ mod tests {
         }
     }
 
+    #[test]
+    fn test_format_status() {
+        let status = Status::Ok(StatusCode::OK);
+        assert_eq!(ColorFormatter::format_status(&status).trim_start(), "[200]");
```

Comment:
here you do `trim_start()` while in the below tests you don't. Is there a reason for that? Shouldn't we test whitespace here as well?

Reply:
Good point. Changed.